bundle/edu.gemini.p2checker/src/main/java/edu/gemini/p2checker/rules/gmos/GmosRule.java

```diff
@@ -1850,6 +1855,33 @@ public IConfigMatcher getMatcher() {
     };
 
     /**
+     * REL-2456: We request a Phase II check to give a warning if the DTA-X
+     * position is not a multiple of the y-pixel binning.
+     */
+    private static IConfigRule DTA_X_Y_MULTIPLE_BINNING_RULE = new IConfigRule() {
+        private static final String errMsg = "The DTA X offset (%d) should be multiple of the Y-binning (%d)";
+
+        @Override
+        public Problem check(Config config, int step, ObservationElements elems, Object state) {
+            final int y = getYBinning(config).getValue();
+            final int dtaX = getDtaXOffset(config).intValue();
+
+            if (dtaX % y != 0) {
+                final String msg = String.format(errMsg, dtaX, y);
+                return new Problem(WARNING, PREFIX + "DTA_X_Y_MULTIPLE_BINNING_RULE", msg,
+                        SequenceRule.getInstrumentOrSequenceNode(step, elems));
+            }
+
+            return null;
```

Comment:
I don't recall much of P2, is it correct to return `null` if there are no problems?